spec/taxon_breadcrumbs_spec.rb

```diff
@@ -125,8 +125,8 @@
 
         expect(breadcrumbs).to eq(
           breadcrumbs: [
-            { title: "Home", url: "/" },
-            { title: "Parent A", url: "/parent-a" },
+            { title: "Home", url: "/", is_page_parent: false },
+            { title: "Parent A", url: "/parent-a", is_page_parent: true },
             { title: "Taxon", is_current_page: true },
```

Comment:
Can we add a test case where we don't have `is_current_page` ? Sorry, I should have added one of these when I implemented this change. I think it would be good to guard against this case, given that one of the original PRs fell into this trap.